### PR TITLE
fix(heex): self closing tag on same line

### PR DESCRIPTION
**Pull request by leandrocp**
Hi @kivikakk here's a fix for a bug I found running the example in https://github.com/leandrocp/mdex/pull/274

Idk when the next version is coming out but I'll test more edge cases soon, either way please always feel free to tag me on any issue/PR related to Phoenix HEEx.

### CODE REVIEW

File: src/nodes.rs [hunk 1]
```diff
@@ -6,7 +6,7 @@ use std::convert::TryFrom;
 
 use crate::arena_tree;
 #[cfg(feature = "phoenix_heex")]
-pub use crate::parser::phoenix_heex::NodeHeexBlock;
+pub use crate::parser::phoenix_heex::{HeexNode, NodeHeexBlock};
```
Comment (leandrocp) [inline src/nodes.rs]: I had to export `HeexNode` to use on my NIF (to have Elixir/Rust integration)

**Comment by kivikakk** [comment]
Nice! And yes, I will tag you if something comes up and I can't work out it :) MDEx would be most affected and I wouldn't wanna break it by accident!
